smartmin/views.py

```diff
@@ -427,9 +427,9 @@ def get_queryset(self):
 
     def derive_title(self):
         """
-        By default we just return the string representation of our object
+        Fallback to the string representation of our object
         """
-        return str(self.object)
```

Comment:
this is why `title = ` doesn't work in `SmartReadView` and we usually end up having to add the title explicitly in the template